api/groups/internalGroup.go

```diff
@@ -370,6 +410,32 @@ func (ib *internalBlockGroup) getInternalMetaBlockByRound(c *gin.Context) {
 	shared.RespondWith(c, http.StatusOK, gin.H{"block": block}, "", shared.ReturnCodeSuccess)
 }
 
+func (ib *internalBlockGroup) getInternalStartOfEpochMetaBlock(c *gin.Context) {
+	epoch, err := getQueryParamEpoch(c)
+	if err != nil {
+		shared.RespondWithValidationError(
+			c, fmt.Sprintf("%s: %s", errors.ErrValidation.Error(), errors.ErrInvalidEpoch.Error()),
+		)
+		return
+	}
+
+	start := time.Now()
+	block, err := ib.getFacade().GetInternalStartOfEpochMetaBlock(common.ApiOutputFormatInternal, epoch)
```

Comment:
so we have 
```GO
common.ApiOutputFormatInternal
common.ApiOutputFormatProto
```
Can we rename ApiOutputFormatInternal to something else? It took me several minutes to figure out which is returning what. InternalMarshalizer is the GogoProtoMarshalizer. I thought they were the same.

Can be done in a separate PR

Reply:
will do in a separate PR :+1: